sources/packages/api/src/services/education-program-offering/education-program-offering-import-csv.models.ts

```diff
@@ -0,0 +1,296 @@
+import { Type } from "class-transformer";
+import {
+  ArrayMinSize,
+  IsDateString,
+  IsEnum,
+  IsIn,
+  IsNotEmpty,
+  IsNumber,
+  IsOptional,
+  Matches,
+  ValidateIf,
+  ValidateNested,
+} from "class-validator";
+import { removeDoubleWhiteSpaces } from "../../utilities";
+import { OfferingIntensity } from "../../database/entities";
+import {
+  currencyNumberOptions,
+  OfferingDeliveryOptions,
+} from "./education-program-offering-validation.models";
+
+/**
+ * Expected date format to be received from the CSV.
+ */
+const DATE_FORMAT = "YYYY-MM-DD";
+
+/**
+ * For study breaks periods, this is the place holder to receive the array index.
+ */
+export const STUDY_BREAK_INDEX_PLACE_HOLDER = "{index}";
+
+/**
+ * user-friendly header names used in the CSV to be populated by the user.
+ * The CSV model parser uses this as a base to parse the CSV string into an object model.
+ */
+export const CSVHeaders = {
+  LocationCode: "Institution Location Code",
```

Comment:
Why didn't we use  camel case here for the object properties?

Reply:
At first, it was an enum, and then it was changed. It can be changed to camel case. I am not sure which is better. Thoughts?

Reply:
Changed it to camel case.